declarations/WebpackOptions.d.ts

```diff
@@ -1040,7 +1040,7 @@ export interface WebpackOptions {
  */
 export interface MemoryCacheOptions {
 	/**
-	 * Additionally cache computation of modules that are unchanged and reference only unchanged modules.
+	 * Additionally cache computation of modules that are unchanged and reference only unchanged modules in memory.
```

Comment:
I didn't change this line. This was added automatically by `yarn run fix:special`